src/main/java/com/allthemods/gravitas2/core/mixin/WorldGenFireDragonRoostMixin.java

```diff
@@ -0,0 +1,23 @@
+package com.allthemods.gravitas2.core.mixin;
+
+import com.github.alexthe666.iceandfire.world.gen.WorldGenFireDragonRoosts;
+import net.dries007.tfc.common.blocks.TFCBlocks;
+import net.dries007.tfc.common.blocks.rock.Rock;
+import net.minecraft.world.level.block.Blocks;
+import net.minecraft.world.level.block.state.BlockState;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.Overwrite;
+
+@Mixin(value = WorldGenFireDragonRoosts.class,remap = false)
+public abstract class WorldGenFireDragonRoostMixin {
+
+    /**
+     * @author thevortex
+     * @reason
+     */
+    @Overwrite(remap = false)
+    protected BlockState transform(BlockState state) {
+        if (state.getBlock() == Blocks.COBBLESTONE) { return Blocks.AIR.defaultBlockState(); }
+        return state.getBlock() == Blocks.AIR ? Blocks.AIR.defaultBlockState() : state;
```

Comment:
isn't this check just useless?

Reply:
```
if (state.is(Blocks.COBBLESTONE)) {
  return Blocks.AIR.defaultBlockState();
}
return state;
```

Reply:
I was talking about the `state.getBlock() == Blocks.AIR` part

Reply:
That’s the part I removed…